components/combobox/themes/express.css

```diff
@@ -22,5 +22,7 @@
 		--spectrum-combobox-border-color-focus: var(--spectrum-gray-900);
 		--spectrum-combobox-border-color-focus-hover: var(--spectrum-gray-800);
 		--spectrum-combobox-border-color-key-focus: var(--spectrum-gray-900);
+
+		--spectrum-combobox-readonly-input-border-color: var(--spectrum-gray-400);
```

Comment:
I see one Express difference in the VRT for this change in the read-only border color. I don't see this one mentioned in the Jira or PR/validation; was this change intended/discussed?

Reply:
Oh shoot, I forgot to add that to the validation! Great catch. I did mean to make this change, but it was not discussed.

I stumbled on this by chance. As I was going through a bunch of border colors, I saw the default border color was `gray-500`, and the read-only border color was `gray-500` in both S1 and S2 contexts. The default `express.css` border color was `gray-400`, but because the read-only input variable wasn't defined, I noticed it was still rendering at `gray-500`. It seemed like the intention was the keep the same border color for read-only.

I can add the read-only validation to the PR description if you're good with that logic!

Reply:
Here's what is on production currently:
<img width="511" alt="Screenshot 2025-03-13 at 5 19 57 PM" src="https://github.com/user-attachments/assets/41241799-700d-461e-8b93-c73ff4938a60" />
<img width="510" alt="Screenshot 2025-03-13 at 5 20 03 PM" src="https://github.com/user-attachments/assets/8e718186-6c10-4052-97e4-27678d99e010" />

That just didn't seem right, compared to S1 & S2.

Reply:
That makes sense that it should match the default like the other themes (and not be darker than default). I'd add to the changeset to document that bug fix for Express and the border color that is changing.